tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.UI/Views/ProcessPerformanceTableControl.xaml

```diff
@@ -0,0 +1,111 @@
+                <Grid ColumnSpacing="0">
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="40"/>
+                    </Grid.RowDefinitions>
+                    
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="180" />
+                        <ColumnDefinition Width="130" />
+                        <ColumnDefinition Width="200" />
+                    </Grid.ColumnDefinitions>
+                    
+                    <Border Grid.Column="0" Style="{StaticResource BorderStyle}">
+                        <TextBlock
+                            x:Uid="ProcessPerformanceTableControl_Heading_Process"
+                            Padding="20 0 0 0"
+                            VerticalAlignment="Center"
+                            HorizontalAlignment="Left"
+                            HorizontalTextAlignment="Left" />
+                    </Border>
+                    <Border Grid.Column="1" Style="{StaticResource BorderStyle}">
+                        <TextBlock
+                            x:Uid="ProcessPerformanceTableControl_Heading_ProcessType"
+                            Margin="40 0 0 0"
+                            VerticalAlignment="Center"
+                            HorizontalAlignment="Left"
+                            HorizontalTextAlignment="Left" />
+                    </Border>
+                    <Border Grid.Column="2" Grid.ColumnSpan="2" Style="{StaticResource BorderStyle}">
+                        <TextBlock
+                            x:Uid="ProcessPerformanceTableControl_Heading_CpuAboveThreshold"
+                            Margin="40 0 0 0"
+                            VerticalAlignment="Center"
+                            HorizontalAlignment="Left"
+                            HorizontalTextAlignment="Left"
+                            TextTrimming="CharacterEllipsis" />
+                    </Border>
+                </Grid>
+            </ListView.Header>
+            
+            <ListView.ItemTemplate>
+                <DataTemplate x:DataType="local:ProcessData">
+                    <Grid ColumnSpacing="5">
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="180" />
+                            <ColumnDefinition Width="130" />
+                            <ColumnDefinition Width="200" />
+                        </Grid.ColumnDefinitions>
+
+                        <Border Grid.Column="0" Style="{StaticResource BorderStyle}" BorderThickness="0, 0, 1, 0">
+                            <TextBlock
+                                Text="{x:Bind Name}"
+                                VerticalAlignment="Center"
+                                HorizontalAlignment="Left"
+                                HorizontalTextAlignment="Left"
+                                TextTrimming="CharacterEllipsis"
+                                Padding="11 11 11 11" />
```

Comment:
Super nit/personal preference, but sometimes here it's just simpler to say "Padding="11"" (if they're all the same, you can just use one)